bitnami/redis/README.md

```diff
@@ -815,14 +815,27 @@ helm install my-release --set master.persistence.existingClaim=PVC_NAME oci://RE
 | `sentinel.service.externalTrafficPolicy`                     | Redis&reg; Sentinel service external traffic policy                                                                                                                                                                          | `Cluster`                        |
 | `sentinel.service.extraPorts`                                | Extra ports to expose (normally used with the `sidecar` value)                                                                                                                                                               | `[]`                             |
 | `sentinel.service.clusterIP`                                 | Redis&reg; Sentinel service Cluster IP                                                                                                                                                                                       | `""`                             |
-| `sentinel.service.createMaster`                              | Enable master service pointing to the current master (experimental)                                                                                                                                                          | `false`                          |
+| `sentinel.service.createMaster`                              | DEPRECATED. Use sentinel.masterService.createMaster instead                                                                                                                                                                  | `false`                          |
```

Comment:
We should not mark a feature as deprecated while we mark the alternative as experimental. IMO one of them has to be 'stable', so I would mark this still as stable and usable. Also, I think that people may still want to apply configurations just to both services, so we should stay stable here.